packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -525,33 +543,96 @@ export class RemoteMediaManager extends EventsScope {
     this.mediaRequestManagers.video.commit();
   }
 
+  /**
+   * Sets which named media group need receiving
+   * @param {MediaType} mediaType of the stream
+   * @param {number} languageCode of the stream
+   * @returns {void}
+   */
+  public async setReceiveNamedMediaGroup(mediaType: MediaType, languageId: number) {
```

Comment:
how can the client app say with this API that they don't want to receive any si audio anymore?

Reply:
Call the function with the languageId as zero

Reply:
ok, so this needs to be documented in the jsdoc

Reply:
Done